src/kindhandlers/common/formLoader.ts

```diff
@@ -8,9 +8,14 @@ const uiformSchemaCache = new Map<string, any>();
 export function getFormSchema(kind: string) {
   try {
     if (!formSchemaCache.has(kind)) {
+      const metadataSchema = loadResource(`form-schemas${path.sep}metadata-schema.json`);
       const formSchema = loadResource(`form-schemas${path.sep}${kind.toLowerCase()}-schema.json`);
-      if (formSchema) {
-        formSchemaCache.set(kind, JSON.parse(formSchema));
+      if (formSchema && metadataSchema) {
```

Comment:
Can there be cases where the Form schema is available, but the metada not? 🤔

Reply:
I know as all resources should have metadata. I don't know a case that it doesn't have